internal/librarian/librarian_test.go

```diff
@@ -169,3 +174,68 @@ func runGit(t *testing.T, dir string, args ...string) {
 		t.Fatalf("git %v: %v", args, err)
 	}
 }
+
+// setupRepoForGetCommits creates an empty gitrepo and creates some commits and
+// tags.
+//
+// Each commit has a file path and a commit message.
+// Note that pathAndMessages should at least have one element. All tags are created
+// after the first commit.
+func setupRepoForGetCommits(t *testing.T, pathAndMessages []pathAndMessage, tags []string) *gitrepo.LocalRepository {
```

Comment:
I believe go compiles all test files in a package together. It is not clear to me why moving this helper to librarian_test.go? It is not a dedicated helper file, maybe create test helper file.